figs/stanplots/analyses_wanang.R

```diff
@@ -1,31 +1,43 @@
-# Specify a path for data and rds files
-setwd("C:\\Users\\Piotr Szefer\\Desktop\\Work\\garden experiment\\figs\\stanplots")
+
 
 # Examples
 # https://www.sciencedirect.com/science/article/pii/S0095447017302310
 
-# Upload data
-AllTestData <- read.table("all_test_data.txt")
-TreeTestDataset <- read.table("tree_test_data.txt")
+
+# load data and packages --------------------------------------------------
+
+# load data
+AllTestData <- read.table("figs/stanplots/all_test_data.txt")
+TreeTestDataset <- read.table("figs/stanplots/tree_test_data.txt")
 
 # Load package 
 library(brms)
+library(ggplot2)
+library(bayesplot)
+library(tidybayes)
+library(dplyr)
 
+
+# plots of the raw data ---------------------------------------------------
```

Comment:
this is called a code section in rstudio! any comment that ends with multiple dashes. you can actually collapse them, and navigate among them! insert them with Cntrl-shift-r or Code>insert section